src/data/entities/SavedItem.ts

```diff
@@ -16,6 +16,9 @@ export class SavedItem extends BaseEntity {
   @Column()
   third_party_url: string;
 
+  @Column()
+  third_party_resource_id: number;
```

Comment:
En esta clase, y en otras partes del sistema, e.g. `SavedItemService.ts`, se utiliza el término `item` para referirse a un anime o manga. Creo que es mejor ser consiste con eso en lugar de usar `resource`

```suggestion
  third_party_item_id: number;
```

Reply:
List, se renombro la columna de la entidad